pokemon-checker/src/services/AbilityService.ts

```diff
@@ -0,0 +1,19 @@
+import { AbilityFactory } from "../factories/AbilityFactory";
+
+/**
+ * Service for handling fetches related to Abilities, and
+ * their descriptions
+ */
+export class AbilityService {
+
+  private factory: AbilityFactory;
+  //create ability repository
+
+  constructor() {
+    this.factory = new AbilityFactory();
+  }
+
+  public getAbility(url: string): Promise<any> {
+    return fetch(url).then((response) => response.json());
```

Comment:
The service should be using the factory to return `AbilityDTO` data, but this works for now.

Reply:
I'll take a look at this